SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -3320,25 +3320,7 @@ void CSolver::LoadInletProfile(CGeometry **geometry,
 
   unsigned long Marker_Counter = 0;
 
-  bool turbulent = (config->GetKind_Solver() == RANS ||
```

Comment:
We are going to end up with changes to the same functions here and in #1264 maybe you could review that PR since you are also fixing inlet profile issues.